onboard/nucleo_bridge/ControllerMap.cpp

```diff
@@ -0,0 +1,76 @@
+#include "ControllerMap.h"
+#include "Controller.h"
+
+uint8_t ControllerMap::calculate_i2c_address(uint8_t nucleo, uint8_t channel) {

```

Comment:
Please add comments to functions (throughout).

Reply:
Implemented